backend/tracim_backend/models/context_models.py

```diff
@@ -600,20 +600,27 @@ def __init__(
         count: int,
         event_types: Optional[List[EventTypeDatabaseParameters]] = None,
         page_token: Optional[str] = None,
+        exclude_author_id: Optional[int] = 0,
```

Comment:
Should the default value be `None` instead (it gives the same result of course, as long as we do not have user with id equal to 0)?

Reply:
I hesitated, I'm fine with None. I'll change this.